### DISCUSSION TITLE
请问remnote中有没有目录功能啊？

### DISCUSSION THREAD

**Discussion by zmh908264302** [5eagull/RemNote-Chinese-chat-group#48] [你问我答/互相帮助]
如题。

**Comment by zmh908264302** [comment]
@lovedebug

**Comment by sepwolves** [comment]
Allnotes可不可以算是目录呢？

**Comment by MyEnthusiastic** [comment]
我觉得要更好的使用RN，最好丢掉之前的使用习惯，即不要太关注目录功能，好好看文档。

**Comment by makerjackie** [comment]
没有，但是你可以通过快捷键 折叠rem 来实现 "目录"功能，我觉得目录一直是缺失的功能

**Comment by sepwolves** [comment]
我现在就是这么干的。

jackiexiao ***@***.***> 于2022年3月15日周二 23:13写道：

**Comment by sepwolves** [comment]
你说的是：好好看文档。

sfs ***@***.***> 于2022年3月15日周二 22:53写道：

**Reply by Orisik** [reply]
remnote正在考虑造个右侧栏, 应该会有该文档 标题目录 功能
